docs/components/Interval.md

```diff
@@ -0,0 +1,41 @@
+# Interval
+
+The Interval component is used for when it's necessary to re-render at regular intervals. Also known as Frame.
+
+```js
+import { Interval } from 'react-powerplug'
+```
+
+```jsx
+<Interval delay={1000}>
```

Comment:
I don't think `delay` is a good name for interval. For end user it will look like the component will be first rendered after 1s.

Reply:
I do not like it either too much... But I saw that mozilla docs use `delay` for `setInterval`
https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setInterval

Do you have any suggestion? I do not know any better names. :(

Reply:
For setInterval it's actually a delay. The function inside is called after specified time.

Reply:
I don't understand the difference. :confused: